examples/react/pagination/pages/index.js

```diff
@@ -35,7 +35,7 @@ function Example() {
 
   // Prefetch the next page!
   React.useEffect(() => {
-    if (data?.hasMore) {
+    if (!isPreviousData && data?.hasMore) {
```

Comment:
if we do that, we need to add `isPreviousData` to the effect's dependency array.

Reply:
I will update my PR, I am still new to this.